src/PrestaShopBundle/Resources/config/services/bundle/translation.yml

```diff
@@ -108,23 +108,37 @@ services:
         tags:
             - {name: translation.loader, alias: db}
 
+    prestashop.translation.legacy_file_reader:
+        class: PrestaShopBundle\Translation\Loader\LegacyFileReader
+        arguments:
+            - "@prestashop.core.translation.locale.converter"
+
     prestashop.translation.legacy_file_loader:
       class: PrestaShopBundle\Translation\Loader\LegacyFileLoader
       arguments:
-        - "@prestashop.core.translation.locale.converter"
+          - "@prestashop.translation.legacy_file_reader"
       tags:
           - {name: translation.loader, alias: legacy_files}
 
+    # cannot decorate the original one because it is loaded later
+    # this version activates loading legacy wordings in smarty files
+    prestashop.translation.extractor.smarty.legacy:
+        class: PrestaShop\TranslationToolsBundle\Translation\Extractor\SmartyExtractor
+        arguments:
+            - "@prestashop.compiler.smarty.template"
+            - true
```

Comment:
Yes you can still use the legacy service, which is called `prestashop.translation.extractor.smarty`. The idea was to keep the old behavior intact.

Keep in mind that we use the same TranslationToolsBundle in our internal TranslationTool to extract core wordings (aka without `mod=`) which we then push to Crowdin. We don't want legacy translations in that extractor.

Reply:
Then couldn't we include them by default (which would avoid adding an extra service in the core) And update the TranslationTool so that it doesn't extract legacy translations.
Although I get that you don't want to change the initial behavior it just feels weird to have to redefine a bundle service just to toggle a behavior

Reply:
The normal behavior is not to extract legacy translations. This is a "temporary hack" if you will, as the legacy system is meant to be removed. 

I don't think redeclaring the bundle service is that big of a deal. At least the behavior is self-explanatory :)

Reply:
Sure, but "temporary" has a tendency of becoming "everlastingly" :D
But this should do it for now!